src/com/amazon/corretto/crypto/provider/DebugFlag.java

```diff
@@ -23,7 +23,12 @@ enum DebugFlag {
      * Increases the verbosity of logs.
      * May still need to be combined with increasing the log level of your configured logger.
      */
-    VERBOSELOGS;
+    VERBOSELOGS,
+    /**
+     * By default ACCP attempts to delete the native libraries it uses immediately after loading.
+     * When this value is set it does not delete them until the JVM is done running.
```

Comment:
as in, unlink them on the filesystem? if the JVM process is restarted, wouldn't the libraries be gone?

also, why would we want to leave them around in the debug case?

Reply:
Yes, as in unlink them from the filesystem.

Without the flag the flow looks like this:
1. Write files to disk
2. Load libraries from files
3. Try to delete files from disk (but don't worry if you fail)
4. (JVM Shutdown) Automatically delete files if still thre

This flag causes step 3 above to be skipped so that the files live until the JVM terminates. This makes it easier to do *native* debugging (say, with GDB) because your debugger can actually see the files which contain the running code.